src/CSS/Profile.css

```diff
@@ -21,17 +18,40 @@
 .profile-image-bubble > img {
     object-fit: cover;
 }
+.profile-image {
+    border-radius: 100%;
+    object-fit: cover;
+    height: 100%;
+}
+
+.profile-navigation {
+    width:80%;
+    max-width:280px;
+    margin:0 auto;
+    display:flex;
+    flex-direction:column;
+    align-items: flex-start;
+}
+
+.profile-navigation > a {
+    margin:5px 0;
+}
+.profile-navigation > .text-link-warning {
+    margin-top:25px;
+}
+
+
 
 .settings-card {
-    background-color: var(--color-green-lighter);
+    /*background-color: var(--color-green-lighter);*/
```

Comment:
Feel free to remove these settings entirely. I'd rather re-write a few lines than accruing zombie code over time.